pages/common/rtmpdump.md

```diff
@@ -0,0 +1,20 @@
+# rtmpdump
+
+> A tool to dump media content streamed over the RTMP protocol.
+> More information: <http://rtmpdump.mplayerhq.hu/>.
+
+- Download a file:
+
+`rtmpdump --rtmp {{rtmp://example.com/path/to/video}} -o {{file.ext}}`
+
+- Download a file from a Flash player:
+
+`rtmpdump --rtmp {{rtmp://example.com/path/to/video}} --swfVfy {{http://example.com/player}} --flashVer {{LNX 10,0,32,18}} -o {{file.ext}}`
```

Comment:
Does the argument to `--flashVer` not need double quotes around it?

Reply:
Oh right, I added them.